### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by Golem642**
Added fix for two major french school websites (Toutatice and Pronote)

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -971,15 +971,6 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
-aosogrenci.anadolu.edu.tr
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:974]: Revert

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -5986,13 +5977,6 @@ INVERT
 
 ================================
 
-flashscore.com.tr
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5989]: revert

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -12725,6 +12709,26 @@ body {
...
+.Cours
+.Insecable
+.Calendrier_Jour_Selection
+
+CSS
+body,
+.ie-chips.tag-style {
+	border-radius: .1rem;
+	padding-right: 0;
+	position: relative;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:12725]:
```suggestion
```

Don't think these are needed.

Comment (Golem642) [reply src/config/dynamic-theme-fixes.config]: You're right, i will also delete the background-color as it is useless, i just tested

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -12725,6 +12709,26 @@ body {
...
+	background-color: transparent !important;
```
Comment (Golem642) [inline src/config/dynamic-theme-fixes.config:12723]:
```suggestion
```
Removing this line because it's useless

**Review by Golem642 — COMMENTED** [review]
Removed useless lines that i thought needed to be here

**Comment by Golem642** [comment]
Sorry, i messed up. I though i applied the suggestions but i didn't, i will create another pull request with modifications

**Pull request closed without merging**